src/sentry/static/sentry/app/components/charts/percentageBarChart.jsx

```diff
@@ -0,0 +1,131 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import moment from 'moment';
+
+import BarSeries from './series/barSeries.jsx';
+import BaseChart from './baseChart';
+import Tooltip from './components/tooltip';
+import XAxis from './components/xAxis';
+import YAxis from './components/yAxis';
+
+const FILLER_NAME = '__filler';
+
+/**
+ * A stacked 100% column chart over time
+ *
+ * See https://exceljet.net/chart-type/100-stacked-bar-chart
+ */
+export default class PercentageBarChart extends React.Component {
+  static propTypes = {
+    ...BaseChart.propTypes,
+
+    series: PropTypes.arrayOf(
+      PropTypes.shape({
+        seriesName: PropTypes.string,
+        data: PropTypes.arrayOf(
+          PropTypes.shape({
+            category: PropTypes.string,
+            value: PropTypes.number,
+          })
+        ),
+      })
+    ),
+
+    getCategoryName: PropTypes.func,
```

Comment:
I'm not fully clear on the purpose of this function. Since category is already being passed in as part of the series above, why do we need to expose a function allowing someone to change the calculation of that?